### PR TITLE
docs: Extend section on how to collect WinEvent logs

### PR DESCRIPTION
**Pull request by LorbusChris**

### CODE REVIEW

**Review by aravindhp — APPROVED** [review]
/approve

@codyhoag we can merge this on your lgtm.

File: docs/TROUBLESHOOTING.md [hunk 1]
```diff
@@ -58,12 +58,24 @@ ip-10-0-138-252.us-east-2.compute.internal pods/
...
-The Windows Docker service does not log to stdout but instead logs to the Windows' event log. You can view the Docker
-event logs using the following steps:
+The Get-WinEvent shim on the kubelet logs endpoint can be used to collect application event logs from Windows machines.
+E.g. for the docker runtime service:
+```shell script
+$ oc adm node-logs -l kubernetes.io/os=windows --path=journal -u docker
+```
+The same command is executed when collecting logs with `oc adm must-gather`.
+
+Other Windows application logs from the EventLog can also collected by specifying the respective service on a `-u` flag.
```
Comment (codyhoag) [inline docs/TROUBLESHOOTING.md:72]:
```suggestion
Other Windows application logs from the EventLog can also be collected by specifying the respective service on a `-u` flag.
```

**Review by codyhoag — APPROVED** [review]
Just one nit; lgtm otherwise! I'll add this to the official doc PR today :+1:

File: docs/TROUBLESHOOTING.md (hunk 1 above)
Comment (LorbusChris) [reply docs/TROUBLESHOOTING.md]: thanks! Updated

**Pull request merged**